docs/ops/sort/TopK_3.md

```diff
@@ -10,7 +10,7 @@
 
```

Comment:
Please, add a "Detailed description section" where all combinations of values for "mode" and "sort" will be explicitly described (like it is done in the ticket).